packages/diskuvbox/diskuvbox.0.1.0/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Cross-platform basic set of script commands"
+description:
+  "A cross-platform basic set of script commands. Available as a single binary (`diskuvbox`, or `diskuvbox.exe` on Windows) and as an OCaml library."
+maintainer: ["opensource+diskuv-ocaml@support.diskuv.com"]
+authors: ["Diskuv, Inc. <opensource+diskuv-ocaml@support.diskuv.com>"]
+license: "Apache-2.0"
+homepage: "https://github.com/diskuv/diskuvbox"
+bug-reports: "https://github.com/diskuv/diskuvbox/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "odoc" {>= "1.5.3" & with-doc}
+  "ocaml" {>= "4.09.0"}
+  "ppx_deriving" {>= "5.2.1"}
+  "bos" {>= "0.2.0"}
+  "fmt" {>= "0.8.9"}
+  "logs" {>= "0.7.0"}
+  "mdx" {>= "2.0.0"}
+  "headache" {>= "1.05" & build}
+  "ocamlformat" {>= "0.19.0" & build}
+  "cmdliner" {<= "1.2.0"}
```

Comment:
Do you already know that the next cmdliner breaks it? If not, I suggest to remove this bound. If a future release of cmdliner breaks the package we will add the bound and inform you
```suggestion
  "cmdliner"
```

Reply:
I'll replace the bound in dune-project so it doesn't get overwritten